src/editorconfig.rs

```diff
@@ -0,0 +1,316 @@
+#[cfg(feature = "editorconfig")]
+use {
+    crate::{CallParenType, CollapseSimpleStatement, Config, IndentType, LineEndings, QuoteStyle},
+    ec4rs::property::{EndOfLine, IndentSize, IndentStyle, MaxLineLen},
+    std::path::Path,
+};
+
+#[cfg(feature = "editorconfig")]
+pub fn load_editorconfig(config: Config, directory: &Path) -> Config {
+    // HACK: used to detect filetype, this is not a real file path or glob
+    let path = directory.join("*.lua");
+
+    // Read the EditorConfig files that would apply to a file at the given path
+    let mut cfg = ec4rs::properties_of(path).unwrap();
+
+    cfg.use_fallbacks();
```

Comment:
Not sure about this:
https://docs.rs/ec4rs/latest/ec4rs/struct.Properties.html#method.use_fallbacks
https://docs.rs/ec4rs/latest/src/ec4rs/fallback.rs.html